spec/02-integration/04-admin_api/12-plugins-conf_spec.lua

```diff
@@ -31,6 +35,23 @@ describe("Plugins conf property" , function()
       assert.equal(tablex.size(bundled_plugins),
                    tablex.size(json.plugins.available_on_server))
     end)
+    it("expect all plugins are in bundled", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/",
+      })
+      local body = assert.res_status(200 , res)
+      local json = assert(cjson.decode(body))
+      local bundled_plugins_list = json.plugins.available_on_server
+      local rocks_installed_plugins, err = utils.execute("ls -1 $(luarocks show kong | grep 'kong.plugins' | grep -E '/.+/kong/plugins/' -o | uniq)")
+      assert.is_nil(err)
+      local rocks_installed_plugins_list = stringx.split(rocks_installed_plugins, "\n")
+      for _, plugin in ipairs(rocks_installed_plugins_list) do
+        if not not_bundled_plugins[plugin] then
+          assert.truthy(bundled_plugins_list[plugin])
```

Comment:
Let's add a comment or error message here telling the developer that adding the plugin name into `non_bundled_plugins` can silence this error. Otherwise people will need to dig through the test code to find out.

Reply:
Done, please review again, thanks!